templates/createTask.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html>
```

Comment:
please use template inheritance, make a generic template and fill in body in each view.